definitions/checks/disk/available_space_postgresql13.rb

```diff
@@ -0,0 +1,42 @@
+module Checks
+  module Disk
+    class AvailableSpacePostgresql13 < ForemanMaintain::Check
+      metadata do
+        label :available_space_for_postgresql13
+        description 'Check to make sure PostgreSQL 13 work directory has enough space for upgrade'
+        confine do
+          feature(:instance).postgresql_local?
+        end
+      end
+
+      def run
+        assert(psql_13_available_space >= psql_12_consumed_space, warning_message, :warn => true)
```

Comment:
I am worried about warning being strong enough here. If there is not enough space, the upgrade will out right fail won't it?

Reply:
Yeah, will change.